.github/workflows/ci.yml

```diff
@@ -116,6 +116,25 @@ jobs:
         # Please remember to update the CI Schedule Workflow when we add a new version.
         k8s: [ v1.25.0, v1.26.0, v1.27.3 ]
     steps:
+      # Clean unnecessary files to save disk space
+      - name: clean unncessary files to save space
+        run: |
+          df -hl
+          docker rmi `docker images -q`
+          sudo rm -rf /usr/share/dotnet /etc/mysql /etc/php /etc/sudo apt/sources.list.d
+          sudo apt -y autoremove --purge
+          sudo apt -y autoclean
+          sudo apt clean
```

Comment:
I find all the related things could be found in `jlumbroso/free-disk-space@main` in this step, can we delete it?

https://github.com/jlumbroso/free-disk-space/blob/main/action.yml

Reply:
yes, you are absolutely right